lib/internal/blob.js

```diff
@@ -397,25 +396,27 @@ class Blob {
   }
 }
 
-function ClonedBlob() {
-  return ReflectConstruct(function() {
+class CloneableBlob extends Blob {
+  static {
     markTransferMode(this, true, false);
```

Comment:
This line and the following `this[kDeserialize]` should be removed since the `CloneableBlob` constructor itself should not be marked as cloneable. Instances of `CloneableBlob` are marked as cloneable by extending `Blob`, in which the Blob constructor calls `markTransferMode` on the blob instance.

Reply:
The only think I don't understand is: why we need to define `kDeserialize` as `empty fn` but for this case we can use the original implementation?

Reply:
Well, actually the empty `kDeserialize` is never reachable because `ClonedBlob` creates objects by ReflectConstruct from the Blob constructor so the prototype of `ClonedBlob` is not used.